src/ruby/yast/yast.rb

```diff
@@ -215,4 +215,21 @@ def self.import(mname)
 
     base.const_set(modules.last, m) unless base.constants.include?(modules.last.to_sym)
   end
+
+  # Configure global environment for YaST
+  #
+  # Currently it only sets values for $PATH.
+  #
+  # By configuring $PATH, it ensures that correct external programs are executed when
+  # relative paths are given, so possible CVEs are avoided when running YaST. For example:
+  #
+  #   $ export PATH=./:$PATH
+  #   $ ruby -e "system('my_program')"                  # this could execute ./my_program
+  #   $ ruby -e "require 'yast'; system('my_program')"  # this only executes 'my_program' from safe paths
```

Comment:
to be honest I do not much like this magic that simple require of yast will modify PATH which can be intentionally different. I think it is not job of library and instead really job of initial master process. So I would expect it at https://github.com/yast/yast-ruby-bindings/blob/master/src/y2start/y2start#L17 and this method at https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/y2start_helpers.rb

This way when yast is started it sets proper ENV, but not if someone outside just require "yast" ruby bindings.